pkg/operator/controller/gatewayapi/controller.go

```diff
@@ -25,7 +27,10 @@ import (
 )
 
 const (
-	controllerName = "gatewayapi_controller"
+	controllerName                    = "gatewayapi_controller"
+	experimantalGatewayAPIGroupName   = "gateway.networking.x-k8s.io"
+	crdAPIGroupIndexFieldName         = "crdAPIGroup"
```

Comment:
Aligning these two variable names would be helpful.
```suggestion
	gatewayCRDAPIGroupIndexFieldName         = "crdAPIGroup"
```

Reply:
The indexer indexes by the crd group, which can be `gateway` or `not gateway`. So, `crdAPIGroupIndexFieldName` reflects this purpose.

Reply:
Changed the name to `gatewayAPICRDIndexFieldName` which will used in the index with `unmanaged` value.